etc/DependencyInstaller.sh

```diff
@@ -760,6 +778,9 @@ while [ "$#" -gt 0 ]; do
             fi
             export PREFIX="$(realpath $(echo $1 | sed -e 's/^[^=]*=//g'))"
             ;;
+        -save-deps-prefixes=*)
+            saveDepsPrefixes=$(realpath ${1#-save-deps-prefixes=})
+            ;;
```

Comment:
Would it make more sense to always save such file using a reasonable default name and plugging the file into `etc/Build.sh` so that it is always used?

Reply:
That was the final goal to use it by default, but I implemented it in ORFS: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/2295

As it makes more sense to have it here, I'll add sane default and just reuse it from ORFS

Reply:
I've set the default path for the file to `openroad_deps_prefixes.txt` in the same directory as `DependencyInstaller` (usually `etc/openroad_deps_prefixes.txt`).

As building Docker image is multi-stage and `DependencyInstaller` is run from `/tmp`, different default path was chosen for Docker images - `/etc/openroad_deps_prefixes.txt`, which can be changed with argument from `DockerHelper` script.

`Build.sh` search for the file in OpenROAD's `etc` or system `/etc` directory or path to the file can be specify with `-deps-prefixes-file`